backend/inspirehep/records/marshmallow/literature/common/reference_item.py

```diff
@@ -82,9 +81,9 @@ def get_reference_record_id(self, data):
 
     def get_resolved_references_by_control_number(self, data):
         data = force_list(data)
-        resolved_records = InspireRecord.get_linked_records_from_dict_field(
-            data, "record"
-        )
+        from inspirehep.records.api.literature import LiteratureRecord
```

Comment:
Why we need to import it in the function?

Reply:
Circular Imports :/

Reply:
```from inspirehep.records.api import LiteratureRecord``` also?

Reply:
yes